sidebars.js

```diff
@@ -1,4 +1,139 @@
+      ]
+    },
+    {
+      type: 'category',
+      label: 'Searching Algorithms',
+      items: [
+        'searching-algorithms/searching-algorithms',
+        'searching-algorithms/linear-search'
+      ],
+    },
+    {
+      type: 'category',
+      label: 'Sorting Algorithms',
+      items: [
+        'sorting-algorithms/sorting-algorithms',
+        'sorting-algorithms/bubble-sort'
+      ],
+    },
+    {
+      type: 'category',
+      label: 'Data Structures',
+      items: [
+        'data-structures/data-structures',
+        'data-structures/Sorting_0s_1s_2s',
+      ],
+    },
+    {
+      type: 'category',
+      label: 'Dynamic Programming',
+      items: [
+        'dynamic-programming/dynamic-programming',
+        'dynamic-programming/egg-dropping-problem'
+      ],
+    },
+    {
+      type: 'category',
+      label: 'Computational Geometry',
+      items: [
+        'computational-geometry/computational-geometry',
+        'computational-geometry/jarvis-algorithm'
+      ],
+    },
+    {
+      type: 'category',
+      label: 'Mathematics',
+      items: [
+        'mathematics/mathematics',
+        'mathematics/armstrong-number'
+      ],
+    },
+    {
+      type: 'category',
+      label: 'Bit Manipulation',
+      items: [
+        'bit-manipulation/bit-manipulation',
+        'bit-manipulation/brian-kernighan-algorithm'
+      ],
+    },
+    {
+      type: 'category',
+      label: 'Graph Algorithms',
+      items: [
+        'graph-algorithms/graph-algorithms',
+        'graph-algorithms/breadth-first-search'
+      ],
+    },
+    {
+      type: 'category',
+      label: 'Competitive Programming',
+      items: [
+        'competitive-programming/competitive-programming',
+        'competitive-programming/anagram-numbers'
+      ],
+    },
+    {
+      type: 'category',
+      label: 'Machine Learning',
+      items: [
+        'machine-learning/machine-learning',
+        'machine-learning/linear-regression'
+      ],
+    },
+    {
+      type: 'category',
+      label: 'Cryptography',
+      items: [
+        'cryptography/cryptography',
+        'cryptography/caesar-cipher'
+      ],
+    },
+    {
+      type: 'category',
+      label: 'Backtracking',
+      items: [
+        'backtracking/backtracking',
+        'backtracking/n-queens'
+      ],
+    },
+    {
+      type: 'category',
+      label: 'Operating Systems',
+      items: [
+        'operating-systems/operating-systems',
+        'operating-systems/banker-algorithm'
+      ],
+    },
+    {
+      type: 'category',
+      label: 'Miscellaneous',
+      items: [
+        'miscellaneous/miscellaneous',
+        'miscellaneous/sudoku-solver'
+      ],
+    },
+	  {
+      type: 'category',
+      label: 'Docusaurus Tutorial',
+      items: [
+        'getting-started',
+        'create-a-page',
+        'create-a-document',
+        'create-a-blog-post',
+        'markdown-features',
+        'thank-you',
+      ],
+    },
+  ],
```

Comment:
That's not how you do it. You simply duplicated the JSON entry which would not work. Please verify on your local machine before pushing anything on the PR.